src/server.py

```diff
@@ -44,10 +42,13 @@
     "sdr": "sdr.sdr",
     "seasonal_water_yield": "seasonal_water_yield.seasonal_water_yield",
     "urban_flood_risk_mitigation": "urban_flood_risk_mitigation",
+    "urban_cooling_model": "urban_cooling_model",
```

Comment:
I could be wrong about this, but this reminds me a lot of the `_MODEL_UIS` (or some name like that) dict in the CLI file, and it could make sense to reduce the duplication here since this is on the python side of things.  Would that make sense for this case?

Reply:
Yes, I think that would work!